AndroidSDKTests/src/test/java/com/leanplum/internal/ShadowOperationQueue.java

```diff
@@ -16,4 +16,9 @@ public boolean addOperation(Runnable operation) {
     public void addParallelOperation(Runnable operation) {
         operation.run();
     }
+
+    @Override
+    public void addUiOperation(Runnable operation) {
```

Comment:
does this need to be running on the main thread? Is that something that we can make happen?

Reply:
yeah, all public callbacks are executed on main thread